src/appmixer/jira/issues/GetIssue/component.json

```
@@ -1,6 +1,6 @@
 {
     "name": "appmixer.jira.issues.GetIssue",
-    "description": "When triggered, retrieves an issue.",
+    "description": "Gets a selected Issue using its ID..",
```

```suggestion
    "description": "Get a selected Issue using its ID..",
```

To make it consistent with the rest.